crates/pallets/starknet/src/lib.rs

```diff
@@ -167,6 +167,8 @@ pub mod pallet {
         type ChainId: Get<Felt252Wrapper>;
         #[pallet::constant]
         type MaxRecursionDepth: Get<u32>;
+        #[pallet::constant]
+        type ProgramHash: Get<Felt252Wrapper>;
```

Comment:
I'm not sure about that. The StarknetOS is executed outside of the runtime. And I don't see reason of making the sequencer aware of it.